stylesheets/scss/_layout.scss

```diff
@@ -2,3 +2,45 @@
 	Layout goes in this file. Susy setup should go here too.
 	Layout classes should be prefixed `.l-`
 */
+
+// Main Configuration options for the Susy Grid
+$gutter-width: 20px;
+$column-count: 12;
```

Comment:
One thing I find really useful is setting a variable for this called `$grid-cols-full`, then I can switch at a small breakpoint to `$grid-cols-small`. For me in my container that looks like this:
```
	@media (max-width: $breakpoint-small) {
  		@include container($grid-cols-small);
 		max-width: $breakpoint-small;
  	}
```